instrumentation/opentelemetry-instrumentation-botocore/tests/test_botocore_bedrock.py

```diff
@@ -1553,32 +1615,40 @@ def invoke_model_tool_call(
     assert_message_in_logs(
         logs[6], "gen_ai.user.message", user_message_body, span_1
     )
+    if "anthropic.claude" in llm_model_value:
+        choice_content = response_1_body["content"]
+    else:
+        choice_content = response_1_body["output"]["message"]["content"]
     choice_body = {
         "index": 0,
         "finish_reason": "end_turn",
         "message": {
             "role": "assistant",
-            "content": [
-                {
-                    "type": "text",
-                    "text": "Thank you for providing the weather information. Here's the current weather for Seattle and San Francisco:\n\n1. Seattle: The current weather in Seattle is 50 degrees and raining.\n2. San Francisco: The current weather in San Francisco is 70 degrees and sunny.\n\nAs you can see, there's quite a difference in the weather between these two West Coast cities today. Seattle is experiencing cooler temperatures with rain, which is fairly typical for the Pacific Northwest. On the other hand, San Francisco is enjoying warmer temperatures with sunny skies, making for a pleasant day in the Bay Area.\n\nIs there anything else you'd like to know about the weather in these cities or any other locations?",
-                }
-            ],
+            "content": choice_content,
         },
     }
     if not expect_content:
         choice_body["message"].pop("content")
     assert_message_in_logs(logs[7], "gen_ai.choice", choice_body, span_1)
 
 
+@pytest.mark.parametrize(
+    "model_family",
+    ["amazon.nova", "anthropic.claude"],
+)
 @pytest.mark.vcr()
 def test_invoke_model_with_content_tool_call(
     span_exporter,
     log_exporter,
     bedrock_runtime_client,
     instrument_with_content,
+    model_family,
 ):
-    llm_model_value = "us.anthropic.claude-3-5-sonnet-20240620-v1:0"
+    if model_family == "amazon.nova":
+        llm_model_value = "amazon.nova-micro-v1:0"
+    elif model_family == "anthropic.claude":
+        llm_model_value = "us.anthropic.claude-3-5-sonnet-20240620-v1:0"
```

Comment:
consider adding `llm_model_value` to the parameterization to avoid this check

Reply:
I keep the family to keep the cassettes names readable

Reply:
You can do it with pytest.param, but it's up to you (feel free to resolve) https://docs.pytest.org/en/stable/example/parametrize.html#set-marks-or-test-id-for-individual-parametrized-test

Reply:
I'll an entry in the improvements lists since need to check if that would feasible for other tests too, thanks.